contracts/exchange/Orderbook.sol

```diff
@@ -80,6 +84,15 @@ contract Orderbook is IOrderbook, ManagerBase {
         }
     }
 
+    function verifyOrderExists(
```

Comment:
You don't need verifyOrderExists(). Orderbook already has an "exists(order)" function that is public that you can call